compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -698,14 +740,29 @@ def IREELinalgExt_OnlineAttentionOp : IREELinalgExt_PureOp<"online_attention",
     AffineMap getValueMap() {
       return getIndexingMapsArray()[2];
     }
+    std::optional<AffineMap> getMaskMap() {
+      if (getMask()) {
+        return cast<AffineMap>(getIndexingMapsArray()[3]);
+      }
+      return std::nullopt;
+    }
```

Comment:
Can you update the index to be incremented by 1 for mask, output, max, and sum. This is because we introduce map[3] as the scale map now.

Reply:
Great!